app/views/mfa_confirmation/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<% title t('titles.passwords.confirm') %>
```

Comment:
Not sure if it's just because draft, but noting that we should probably want a unique title here from "Confirm the password for your account" ?